src/Illuminate/Routing/CompiledRouteCollection.php

```diff
@@ -229,10 +229,13 @@ protected function newRoute(array $attributes)
         if (empty($attributes['action']['prefix'] ?? '')) {
             $baseUri = $attributes['uri'];
         } else {
+            $prefixParts = trim($attributes['action']['prefix'], '/');
+            $prefixParts = $prefixParts !== '' ? explode('/', $prefixParts) : [];
+
             $baseUri = trim(implode(
                 '/', array_slice(
                     explode('/', trim($attributes['uri'], '/')),
-                    count(explode('/', trim($attributes['action']['prefix'], '/')))
+                    count($prefixParts)
```

Comment:
```suggestion
                    count($prefixParts !== '' ? explode('/', $prefixParts) : [])
```

Reply:
`$prefixParts` is not used anywhere else. I would think this is good enough.

Reply:
Done. Thanks.